frontend/public/components/filter-toolbar.tsx

```diff
@@ -49,31 +52,20 @@ type FilterKeys = {
   [key: string]: string;
 };
 
-const getDropdownItems = (rowFilters: RowFilter[], selectedItems, data, props) =>
-  rowFilters.map((grp) => {
-    const items = grp.itemsGenerator ? grp.itemsGenerator(props, props.kind) : grp.items;
-    return (
-      <SelectGroup key={grp.filterGroupName} label={grp.filterGroupName}>
-        {_.map(items, (item) => {
-          return (
-            <SelectOption
-              data-test-row-filter={item.id}
-              key={item.id}
-              inputId={item.id}
-              value={item.id}
-            >
-              <span className="co-filter-dropdown-item__name">{item.title}</span>
-              <Badge key={item.id} isRead>
-                {grp.isMatch
-                  ? _.filter(data, (d) => grp.isMatch(d, item.id)).length
-                  : _.countBy(data, grp.reducer)?.[item.id] ?? '0'}
-              </Badge>
-            </SelectOption>
-          );
-        })}
-      </SelectGroup>
-    );
-  });
+const RowFilterSelectGroup: React.FC<RowFilterSelectGroupProps> = ({ rowFilter, data }) => (
+  <SelectGroup key={rowFilter.filterGroupName} label={rowFilter.filterGroupName}>
+    {_.map(rowFilter.items ?? [], (item) => (
```

Comment:
nit: prefer optional chaining. (I think the `?? []` is unnecessary if using `_.map` as well since it handles null/undefined.)